textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/AlchemyAPIexample/requirements.txt

```diff
@@ -0,0 +1,9 @@
+To use AlchemyAPI you need to make sure you have:
```

Comment:
Can we rename it to "readme.md" and use the markdown syntax?  Check an example in another PR about Lingpipe: https://github.com/TextDB/textdb/pull/262/files